massa-consensus-exports/src/events.rs

```diff
@@ -3,4 +3,6 @@
 pub enum ConsensusEvent {
     /// probable desynchronization detected, need re-synchronization
     NeedSync,
+    /// Network is ended should be send after `end_timestamp`
+    Stop,
```

Comment:
I have updated testnet_19, if you merge it on your branch we shouldn't see this changes anymore.

Reply:
done